src/components/Admin.js

```diff
@@ -22,22 +15,21 @@ class Admin extends React.Component {
     .catch((err) => console.log(err))
   }
 
-  componentDidMount = () => {
-    this.loadAllNiceties((data) => this.setState({niceties: data})
-    );
-  }
+  const [adminNiceties, setAdminNiceties] = useState([]);
+  useEffect(() => {
+    loadAllNiceties((data) => setAdminNiceties(data));
+  }, [])
 
-  render() {
-    return (
+  return (
       <div>
-        {this.state.niceties.map((person) => {
-          let noTextCheck = false;
+        {adminNiceties.map((person) => {
+          let textCheck = false;
           person.niceties.forEach((nicety) => {
             if (nicety.text !== '' && nicety.text !== null) {
-              noTextCheck = true;
+              textCheck = true;
             }
           });
-          if (noTextCheck) {
+          if (textCheck) {
```

Comment:
There's a lot of opportunity for refactoring / code golf here, if you want to get into it, but in the meantime +1 to this clarifying name change!